crates/ruff_linter/src/rules/flake8_datetimez/snapshots/ruff_linter__rules__flake8_datetimez__tests__DTZ005_DTZ005.py.snap

```diff
@@ -1,49 +1,52 @@
 ---
 source: crates/ruff_linter/src/rules/flake8_datetimez/mod.rs
 ---
-DTZ005.py:4:1: DTZ005 The use of `datetime.datetime.now()` without `tz` argument is not allowed
+DTZ005.py:4:1: DTZ005 `datetime.datetime.now()` called without a `tz=` argument
```

Comment:
Nit: 
I think we should either remove `argument` and change it to `tz=<timezone>` or keep the sentence as is but change `tz=` to `tz` because the argument name is `tz` (and not `tz=`). I also thought at first that `tz=` is a typo

```suggestion
DTZ005.py:4:1: DTZ005 `datetime.datetime.now()` called without a `tz` argument
```